packages/contentful--create-contentful-app/src/includeDeliveryFunction.ts

```diff
@@ -55,6 +55,8 @@ export async function cloneDeliveryFunction(destination: string, templateIsTypes
     });
 
     await Promise.all([writeAppManifest, copyBuildFile, writeBuildCommand]);
+
+    await d.remove(deliveryFunctionDirectoryPath, destination, {action: "remove", files: IGNORED_CLONED_FILES});
```

Comment:
```suggestion
    await d.remove(deliveryFunctionDirectoryPath, destination, { action: "remove", files: IGNORED_CLONED_FILES });
```

My OCD is asking can we also remove the space on L58 since we don't have one in the other file? 😅

Reply:
Also - shouldn't the linter have caught the bracket spacing?

Reply:
Maybe it doesn't that those settings 🤷🏾‍♂️